src/web/.htaccess

```diff
@@ -7,6 +7,9 @@ Allow from all
 
 RewriteEngine On
 
+# route openapi resource links from doc; from 13.1 onwards
+RewriteRule ^doc/([1-9]\d|[3-9]\d{2,}[^/]+)/openapi/(.+)$ /openapi/$1/$2 [L,R=302]

```

Comment:
with this crazy regex it only touches versions higher or equals 13 :blush: 
... so the old links remain correct and working

Reply:
good for now